eng/scripts/Cadl-Project-Sync.ps1

```diff
@@ -128,11 +141,42 @@ if ( $configuration["repo"] -and $configuration["commit"]) {
     $specCloneDir = $configuration["spec-root-dir"]
 }
 
-
 $tempCadlDir = "$ProjectDirectory/TempCadlFiles"
 New-Item $tempCadlDir -Type Directory -Force | Out-Null
-CopySpecToProjectIfNeeded `
-    -specCloneRoot $specCloneDir `
-    -mainSpecDir $specSubDirectory `
+CopyToProjectIfNeeded `
+    -cloneRoot $specCloneDir `
+    -mainDir $specSubDirectory `
     -dest $tempCadlDir `
-    -specAdditionalSubDirectories $configuration["additionalDirectories"]
+    -additionalSubDirectories $configuration["additionalDirectories"]
+
+
+# Download the existing SDK
+$loadExisting = $configuration["load-existing"] ?? $true
```

Comment:
My preference is to turn it off by default for now. We have pipeline timeout problems, downloading existing SDK will make things worse.